batchflow/research/named_expr.py

```diff
@@ -24,12 +24,11 @@ def get(self, **kwargs):
             _experiment = experiment
         else:
             _experiment = [experiment]
-        if self.name is not None:
-            res = [item[self.name] for item in _experiment]
-            if len(_experiment) == 1:
-                return res[0]
-            return res
-        return experiment
+        name = self.name if self.name is not None else list(_experiment[0].keys())[0]
```

Comment:
This way of getting a missing name seems rather obscure. Is the first key in the first experiment that is passed to eval_expr is always the name we want? Or they are all equal?

Reply:
Actually, all configs are the same for all units